protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -499,6 +500,15 @@ service TradingDataService {
     // List all entities created by transaction hash
     rpc ListEntities(ListEntitiesRequest) returns (ListEntitiesResponse);
 
+    // Export Network History
+    //
+    // Extract CSV table data from network history between two block heights.
+    // Note: The block heights must fall on network history segment boundaries
+    // which can be discovered with ListAllNetworkHistorySegments
```

Comment:
Because of the way the proto docstrings end up also being the REST docs we can't really reference API by their GRPC name like `ListAllNetworkHistorySegments`.

We're going to have to just be vague and say something like "which can be discovered by listing all network history segments". The plan is to have similar functionality grouped in the docs, so _hopefully_ if you're looking at the doc page for exporting history segments, in the sidebar the docs for the API to list them will be staring you in the face.

Reply:
ah ok I'll change that then :)